### ISSUE TITLE
Permission FREE 4.1.0 Translations in plugin folder

### ISSUE THREAD

**Issue by wocmultimedia** [publishpress/PublishPress-Permissions#1316]
Hi @stevejburge , @richaferry , @andergmartins ,
I installed the new 4.1.0 version and I discovered that the translations ES-FR-IT are already done (at first sight) they match the .pot file's strings. But I haven't done them, so I suppose that you made them with some AI support or other tool. I can state it because I haven't in my archive and also there are some errors such as this "View Knowledge Base" translated as "Visualizza la Knowledge Base". As you can see the string is half Italian and half English, the same for all strings where you set " : ".
No problems, just inform me. I don't fix these strings in the local languages folder. I just update the ones on WP.org today

This is another example of translation not made by me in ES "If you need help or have a new feature request, let us know."
"Si necesita ayuda o tiene alguna solicitud sobre alguna nueva función, háganoslo saber."
That's not Spanish (Spain) language as required by the Spanish community. This is Latin American language used.

The same I found in French language. 
As I stated above, I will correct only the ones on WP.org

**Comment by wocmultimedia** [comment]
Hi @stevejburge ,

No, I'm not sure but those strings were not translated by me and they are the same strings I translated this morning to update files on WP.org.
Anyhow, let me know if I have to fix them and send to you the corrected translations or not

Thanks.

**Comment by wocmultimedia** [comment]
Hi @stevejburge ,

I fixed all issues in the plugin languages folder and here's copies of the files related to v.4.1.0
[PermissionFREE_TranslationUpdates_V.4.1.0.zip](https://github.com/user-attachments/files/18208556/PermissionFREE_TranslationUpdates_V.4.1.0.zip)

**Comment by richaferry** [comment]
Hi @wocmultimedia ,
Thank you for your inspection. I really apreciate it. Yes. Previously i'm already implement your translation as it is. But in time of development we need update the others. So i update the .pot file to scanning new wording to translate. Maybe at that time i run automatic translate using loco translate + google translate engine make some mistake. I saw you're using poedit for translation tool. and i use loco translate. those plugin has different algorythm for sorting. and thats make our changes in git so huge. Is it possible to use loco translate? To avoid mismatch in the future. 
By the way thank you for your new translation update @wocmultimedia 👍

**Comment by wocmultimedia** [comment]
Hi @richaferry ,
The problem isn't Loco that I don't like to use because you must be connected. Then also Google translator is not good for nothing. Besides, the big problem is that Latin American Spanish is quite different than Spanish -Spain. I mostly learned in Perú and at beginning they sent me to hell, so I studied the whole rules of Spanish community and knew what was wrong. Worst with French community they treated me very bad and offensive at beginning but I am stubborn and once more I studied their rules and many other tricks. By the way if do you want save on translation just tell me but please, do not make any mess again. It makes me twice the work and it's a cost in time and money.
I have suggested many times to send me the beta version so I can arrange translations before you make the release. Anyone says ok but then I don't receive the pre-release.

**Comment by richaferry** [comment]
Oh. Okay @wocmultimedia .
Yes. Sure. I'm so sorry for this time. 🙏 
Will inform you next time if need checking for translation

**Comment by wocmultimedia** [comment]
Hi,
There's no problems at all. I'm here to support your team and not only for earning money. I work as they were my products and I like to make a better service every time. People think that is a translation matter only, but it isn't true. I cultivated a good reputation in both Spanish and French communities and also using some oils to the wheel to make the chart going on otherwise translations stay in queue for years.

**Comment by agapetry** [comment]
I agree that we shouldn't be using Google Translate or any other automated tool for this plugin's captions. With so much contextual nuance, that could create a lot of confusion.

**Comment by wocmultimedia** [comment]
@agapetry ,
I just want to point at that is not because I want to make the translations but because how is the situation in WP community today strings are let in waiting if not following the glossaire strictly. This still happen to me even though I know the rules and I have a person which I give a bribe to validate translations. 
Then you can do as you like, but if you make automatic translations I cannot detect these strings at a glance.
